solutions/standard/DA-keys.md

```diff
@@ -0,0 +1,61 @@
+# Configuring keys in a KMS in IBM Cloud projects
+
+When you add a key management service from the IBM Cloud catalog to an IBM Cloud Projects service, you can configure key rings and keys. In the edit mode for the projects configuration, select the Configure panel and then click the optional tab.
+
+In the configuration, specify the name of the key ring, whether the key ring exists, and whether to force the deletion of the key. The object also contains a list of keys with all the information about the keys that you want to create in that key ring.
+
+To enter a custom value, use the edit action to open the "Edit Array" panel. Add the KMS key ring and key configurations to the array here.
+The following example includes all the configuration options for two key rings. One ring contains two keys.
+
+    [
+      {
+        "key_ring_name": "da-ring-1",
+        "existing_key_ring": false,
+        "force_delete_key_ring": true,
+        "keys": [
+          {
+            "key_name": "da-key-1-1",
+            "rotation_interval_month": 1,
+            "standard_key": false,
+            "dual_auth_delete_enabled": false,
+            "force_delete": true
+          },
+          {
+            "key_name": "da-key-1-2",
+            "rotation_interval_month": 1,
+            "standard_key": false,
+            "dual_auth_delete_enabled": false,
+            "force_delete": true
+          }
+        ]
+      },
+      {
+        "key_ring_name": "da-ring-2",
+        "existing_key_ring": false,
+        "force_delete_key_ring": true,
+        "keys": [
+          {
+            "key_name": "da-key-2-1",
+            "rotation_interval_month": 1,
+            "standard_key": false,
+            "dual_auth_delete_enabled": false,
+            "force_delete": true
+          }
+        ]
+      }
+    ]
+
+
+## Key Ring options
```

Comment:
```suggestion
## Configuration options 

Descriptions of the properties in the list.

### Key ring options
```

properties? options? variables?

See my earlier comment about changes to the description and moving this above the example so that it doesn't get lost under it.